src/main/java/org/cactoos/collection/Reversed.java

```diff
@@ -54,16 +55,16 @@ public Reversed(final X... src) {
      */
     public Reversed(final Iterable<X> src) {
         super(
-            new Sticky<>(
-                new CollectionOf<>(
+            new Unchecked<>(
+                new org.cactoos.scalar.Sticky<>(
```

Comment:
@fanifieiev if I'm right, this replacement is also out of scope (`CollectionOf should be deleted`). Maybe it's better to leave a todo?

Reply:
@iakunin Why is out of scope?
The task is to remove `CollectionOf` right? If I remove it I have to put back sth instead right?
I can remove it, but what is supposed to be for the replacement?????

Reply:
@fanifieiev yes, you should put back something. Maybe we should be able to construct `ListOf` from `Scalar`? @victornoel @paulodamaso what do you think?

It is worth noting than org.cactoos.list.ListEnvelope has [ctor from Scalar](https://github.com/yegor256/cactoos/blob/master/src/main/java/org/cactoos/list/ListEnvelope.java#L74).

Reply:
@iakunin see the discussions in #1242 for more context. We are in the collection package, it makes no sense to depend on the `list` package to implement it: the collection package should contain only the strict minimum to decorate collections, but concrete implementations of collection should go in list or set packages.

Also note that ListEnvelope is going to loose its constructor with a `Scalar` in #1255.

Reply:
@victornoel ok, I got it, thank you for your explanation. Closing the conversation.